include/pybind11/iostream.h

```diff
@@ -104,8 +104,10 @@ PYBIND11_NAMESPACE_END(detail)
     .. code-block:: cpp
 
         {
-            py::scoped_ostream_redirect output{std::cerr, py::module_::import("sys").attr("stderr")};
-            std::cerr << "Hello, World!";
```

Comment:
Is this to avoid knows the known flakyness?

Reply:
I think it tripped on this originally and made a mess. But this is a nicer format now.

Reply:
@henryiii `std::cerr << "Hello, World!";` did disappear, though

Reply:
Hmm, not sure what happened, probably a bad merge on my part. It did get updates, so I think the "old" formatting you see here now is better than when the PR started.

Reply:
Pushed a fix to master.

Reply:
Great, thanks!